lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1686,7 +1686,7 @@ def jsonrpc_publish(self, p):
             'txid' : txid of resulting transaction if succesful
             'nout' : nout of the resulting support claim if succesful
             'fee' : fee paid for the claim transaction if succesful
-            'claimid' : claimid of the resulting transaction
+            'claim_id' : claimid of the resulting transaction
```

Comment:
is changing this key safe? it looks like `fee` and `claimid` were not return at all in the past, so they can't be in use anywhere, right?